projects/daq2/Readme.md

```diff
@@ -0,0 +1,9 @@
+# DAQ2 HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad-fmcdaq2-ebz)
+  * Parts : [Quad, 16-Bit, 2.8 GSPS, TxDAC+® Digital-to-Analog Converter](https://www.analog.com/ad9144)
+  * Parts : [14-Bit, 1.25 GSPS/1 GSPS/820 MSPS/500 MSPS JESD204B, Dual Analog-to-Digital Converter](https://www.analog.com/ad9680)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcdaq2-ebz
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcdaq2-ebz/reference_hdl
+  * Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers/iio-dds/axi-dac-dds-hdl https://wiki.analog.com/resources/tools-software/linux-drivers/iio-adc/axi-adc-hdl
```

Comment:
In some cases, when there were more than one linux drivers, they were listed as:
* Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers/iio-dds/axi-dac-dds-hdl
* Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers/iio-adc/axi-adc-hdl

For me it looks better that way